### ISSUE TITLE
My node stopped syncing

### ISSUE THREAD

**Issue by Proxiweb** [input-output-hk/shelley-testnet#102]
**Describe the bug**
My node stopped syncing

**Prerequisites information**
1. jcli 0.5.6 (master-7ab929ea, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
1. jormungandr 0.5.6 (master-7ab929ea, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)]
1. Ubuntu 18.04
1. networking node without explorer with stake pool secret file
1. built from source

**Steps to reproduce the behavior:**
1. Just started the node

**Expected behavior**
I expected my node to continue syncing. It stopped at 18:40

**Additional context**
Oct 08 18:22:40.887 INFO incoming P2P connection closed, peer_addr: 34.74.199.242:34864, task: network
Oct 08 18:26:16.000 INFO leader event starting, date: 228.15381, leader: 1, task: leadership
Oct 08 18:26:16.166 INFO block from leader event successfully stored, date: 228.15381, parent: 3146ba6b9a090b2235983d09bf593ee2c3118f0a03bb19746127d17e54c8747e, hash: 06d77b31b924ac16395a06f05
d14bb512ec3c1a9bac6ded6c09d902850079ee7, task: block
Oct 08 18:26:16.166 INFO update current branch tip, date: 228.15381, parent: 3146ba6b9a090b2235983d09bf593ee2c3118f0a03bb19746127d17e54c8747e, hash: 06d77b31b924ac16395a06f05d14bb512ec3c1a9bac
6ded6c09d902850079ee7, task: block
Oct 08 18:26:16.208 ERRO cannot propagate block to network: send failed because receiver is gone, date: 228.15381, parent: 3146ba6b9a090b2235983d09bf593ee2c3118f0a03bb19746127d17e54c8747e, has
h: 06d77b31b924ac16395a06f05d14bb512ec3c1a9bac6ded6c09d902850079ee7, task: block
Oct 08 18:40:27.304 INFO incoming P2P connection closed, peer_addr: 52.170.44.125:43440, task: network
Oct 08 18:40:30.001 INFO leader event starting, date: 228.15808, leader: 1, task: leadership
Oct 08 18:40:30.162 INFO block from leader event successfully stored, date: 228.15808, parent: 06d77b31b924ac16395a06f05d14bb512ec3c1a9bac6ded6c09d902850079ee7, hash: 0acba07e10b92fc2902b6ee77
209c7c081235897062d2f1045687f67fd5b5585, task: block
Oct 08 18:40:30.162 INFO update current branch tip, date: 228.15808, parent: 06d77b31b924ac16395a06f05d14bb512ec3c1a9bac6ded6c09d902850079ee7, hash: 0acba07e10b92fc2902b6ee77209c7c081235897062
d2f1045687f67fd5b5585, task: block
Oct 08 18:40:30.221 ERRO cannot propagate block to network: send failed because receiver is gone, date: 228.15808, parent: 06d77b31b924ac16395a06f05d14bb512ec3c1a9bac6ded6c09d902850079ee7, has
h: 0acba07e10b92fc2902b6ee77209c7c081235897062d2f1045687f67fd5b5585, task: block
Oct 08 18:40:30.400 INFO incoming P2P connection closed, peer_addr: 85.218.89.33:53692, task: network
Oct 08 18:40:37.272 INFO incoming P2P connection closed, peer_addr: 51.15.106.170:34534, task: network
Oct 08 18:43:29.866 INFO incoming P2P connection closed, peer_addr: 24.27.40.198:47578, task: network
Oct 08 18:54:04.686 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.699 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.712 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.721 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.730 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.740 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.750 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
Oct 08 18:54:04.759 ERRO cannot send PullHeaders request to network: send failed because receiver is gone, task: block
[...same error messages for many lines]

**Comment by Proxiweb** [comment]
It stopped again at 07:01, here is the logs
`Oct 09 07:01:19.127 INFO block successfully applied, date: 228.31438, parent: f1bff7107181b411c03ac5caa71b67e53860016eae115e033522b9dadb267a65, hash: 6c3ae202cd0b19ec1d8910c7e73e574e92baaeb3f785a1cf3fd5a3bb0903cc28, task: block
Oct 09 07:01:19.127 INFO create new branch, date: 228.31438, parent: f1bff7107181b411c03ac5caa71b67e53860016eae115e033522b9dadb267a65, hash: 6c3ae202cd0b19ec1d8910c7e73e574e92baaeb3f785a1cf3fd5a3bb0903cc28, task: block
Oct 09 07:01:19.138 WARN error while streaming response: Error { code: Internal, cause: CannotIterate }, node_id: 401510887185635334, peer_addr: 188.166.36.69:3000, task: network
Oct 09 07:01:19.138 INFO UploadBlocks request failed: Error { code: Internal, source: Status { code: Internal, message: "h2 protocol error: protocol error: unexpected internal error encountered" } }, node_id: 401510887185635334, peer_addr: 188.166.36.69:3000, task: network
Oct 09 07:01:19.636 INFO block successfully applied, date: 228.31444, parent: 6c3ae202cd0b19ec1d8910c7e73e574e92baaeb3f785a1cf3fd5a3bb0903cc28, hash: eaa4b8dee335d3b0e6ced5e77f82a4ffd1071deb181f59e558b11e127f68aacd, task: block
Oct 09 07:01:19.636 INFO create new branch, date: 228.31444, parent: 6c3ae202cd0b19ec1d8910c7e73e574e92baaeb3f785a1cf3fd5a3bb0903cc28, hash: eaa4b8dee335d3b0e6ced5e77f82a4ffd1071deb181f59e558b11e127f68aacd, task: block
Oct 09 07:01:19.644 WARN error while streaming response: Error { code: Internal, cause: CannotIterate }, node_id: 401510887185635334, peer_addr: 188.166.36.69:3000, task: network
Oct 09 07:01:19.644 INFO UploadBlocks request failed: Error { code: Internal, source: Status { code: Internal, message: "h2 protocol error: protocol error: unexpected internal error encountered" } }, node_id: 401510887185635334, peer_addr: 188.166.36.69:3000, task: network
Oct 09 07:36:26.298 INFO Force shutdown worker, 1 connections, scope: global
Oct 09 07:36:26.599 INFO Shutting down node
thread 'leadership1' panicked at 'Expect the event to not close', src/libcore/option.rs:1166:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
Expect the event to not close
Oct 09 07:36:52.719 INFO Starting jormungandr 0.5.6 (master-7ab929ea, release, linux [x86_64]) - [rustc 1.38.0 (625451e37 2019-09-23)], task: init
`

**Comment by dorin100** [comment]
Hi @Proxiweb . The second seems a node crash. In the future, it will help us if before starting the node, you would execute `set RUST_BACKTRACE=full`. This would provide us the entire stack trace of the crash. 

I will open a separate issue in order for our developers to check the above logs. 

Unfortunately, there are still synchronization issues that are tracked here - https://github.com/input-output-hk/jormungandr/issues/860.  More details here - https://github.com/input-output-hk/jormungandr/issues/860#issuecomment-536980736

**Comment by dorin100** [comment]
@Proxiweb - I see that there is another issue opened with the reported crashed - https://github.com/input-output-hk/jormungandr/issues/895

I will close this one as the not syncing issue is tracked by https://github.com/input-output-hk/jormungandr/issues/860
